internal/controller/ansibleRun/ansibleRun.go

```diff
@@ -640,12 +639,11 @@ func (c *connector) acquireAndHoldShard(o controller.Options, s SetupOptions) (i
 	}
 
 AcquireLease:
-	for !acquiredLease {
+	for {
 		for i := 0; i < s.ReplicasCount; i++ {
 			if err := c.acquireLease(ctx, kube, i); err == nil {
 				currentShard = i
 				o.Logger.Debug("acquired lease", "id", i)
-				acquiredLease = true
```

Comment:
Had to remove this to make the linter happy. It's not in use anymore due to how the break uses a tag now